components/automate-ui/src/app/pages/api-token/list/api-token-list.component.ts

```diff
@@ -65,13 +76,40 @@ export class ApiTokenListComponent implements OnInit {
       // Must stay in sync with error checks in create-object-modal.component.html
       name: ['', Validators.required],
       id: ['',
-        [Validators.required, Validators.pattern(Regex.patterns.ID), Validators.maxLength(64)]]
+        [Validators.required, Validators.pattern(Regex.patterns.ID), Validators.maxLength(64)]],
+      projects: [[]]
     });
     this.iamMajorVersion$ = store.pipe(select(iamMajorVersion));
+    this.iamMinorVersion$ = store.pipe(select(iamMinorVersion));
   }
 
   ngOnInit() {
     this.store.dispatch(new GetAllTokens());
+
+    this.iamMajorVersion$
+      .pipe(takeUntil(this.isDestroyed))
+      .subscribe((majorVersion) => {
+        if (majorVersion === null) { return; }
```

Comment:
let's add `filter(identity)` to the pipe instead, it's the thing commonly done, I believe.